R/source.R

```diff
@@ -30,9 +32,10 @@
 #' contains the installed benchmarked package for set-up tasks, and temporarily
 #' remove it during benchmarking with [benchmark_run()] so only one
 #' touchstone library is on the library path at any time.
+#'
 #' @return
-#' The same as [base::source()], which inherits from [base::withVisible()], i.e.
-#' a list with `value` and `visible` (invisibly).
+#'   The same as [base::source()], which inherits from [base::withVisible()], i.e.
```

Comment:
```suggestion
#' The same as [base::source()], which inherits from [base::withVisible()], i.e.
```

Reply:
So if you use 
```r
#' @param x this is 
#'   multi line
```

and 
```r
#' @section x
#' Here no indention.
```

As specified [here](https://style.tidyverse.org/documentation.html#indents-and-line-breaks).